tests/baselines/reference/extractInferenceImprovement.errors.txt

```diff
@@ -32,7 +32,7 @@ tests/cases/compiler/extractInferenceImprovement.ts(28,26): error TS2345: Argume
     // Should fail
     prop = getProperty2(obj, s);
                              ~
-!!! error TS2345: Argument of type 'unique symbol' is not assignable to parameter of type 'never'.
+!!! error TS2345: Argument of type 'typeof s' is not assignable to parameter of type 'never'.

```

Comment:
Hm, this one definitely feels arguable. Not sure if @rbuckton has strong feelings about it, but I don't think it's a blocker.